main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+    "encoding/csv"
+    "fmt"
+    "net/http"
+    "os"
+    "sync/atomic"
+)
+
+var counter uint64
+
+func main() {
+    http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+        fmt.Println("Received request:")
+        writer := csv.NewWriter(os.Stdout)
+        defer writer.Flush()
+        writer.Write([]string{"Header", "Value"})
+        for k, v := range r.Header {
+            for _, val := range v {
+                writer.Write([]string{k, val})
+            }
+        }
+        atomic.AddUint64(&counter, 1)
+    })
+
+    go func() {
+        for {
+            <-time.After(time.Minute)
+            fmt.Printf("Total requests received: %d\n", atomic.LoadUint64(&counter))
```

Comment:
In addition to printing the counter value, please also store the value in a SQLite database.